app/scripts/ChromosomeGrid.js

```diff
@@ -288,7 +288,8 @@ class ChromosomeGrid extends PixiTrack {
     const output = document.createElement('g');
     track.appendChild(output);
 
-    base.setAttribute('id', 'ChromosomeGrid');
+    const cgUniqueUd = Math.random().toString(36).substring(7);
```

Comment:
Same comment as above about using `slugid` for random ids.

Reply:
Added `slugid.nice()` call.